docs/HACKING.md

```diff
@@ -1,104 +1,147 @@
-⚠⚠⚠ THIS IS A LIVING DOCUMENT AND LIKELY TO CHANGE QUICKLY ⚠⚠⚠
-
 # Hacking on the WMCO
-## Pre-requisites
-- [Cluster and OS pre-requisites](wmco-prerequisites.md)
-- [Install](https://sdk.operatorframework.io/docs/installation/) operator-sdk v1.15.0
-- The operator is written using operator-sdk [v1.15.0](https://github.com/operator-framework/operator-sdk/releases/tag/v1.15.0)
-  and has the same [pre-requisites](https://sdk.operatorframework.io/docs/installation/#prerequisites) as it does.
-- Git submodules need to be initialized and up to date before running the build command. Please refer to
-  [updating Git submodules](#updating-git-submodules).
-- Verify push/pull access to a container registry from where the cluster can pull image from.
 
-## User Workflows
+This guide will walk you through the workflows required to run local e2e tests on the WMCO, and perform tasks like 
+update the submodule versions, and deploy custom workloads on a cluster. 
+
+## Cluster Prequisites
+
+The WMCO releases match the versions of OpenShift they are run with. Changes made to a 4.10 release should be tested
+with a 4.10 image and so on.
+Changes made against master should be tested against a nightly version of OpenShift.
+
+- [OpenShift Installer Mirror](<https://mirror.openshift.com/pub/openshift-v4/clients/ocp/>)
+- [OpenShift Nightly Builds](<https://amd64.ocp.releases.ci.openshift.org/#4-dev-preview>)
 
-### I want to install an officially released version of WMCO on my OpenShift/OKD cluster
-* Instructions to install the community operator can be found in the [README](/README.md).
-  Customers with a Red Hat subscription should instead look at the [OpenShift docs.](https://docs.openshift.com/container-platform/latest/windows_containers/enabling-windows-container-workloads.html)
+### Pre-workflow checklist
 
-### I want to try the unreleased WMCO
-* [Build the operator](#build)
-* [Deploy the operator directly](#deploying-the-operator-directly)
+- Download the correct OpenShift installer
+- [Initialize git submodules](<https://github.blog/2016-02-01-working-with-submodules/>)
+- [Verify push/pull access to a container registry such as quay.io](<https://quay.io>)
+- [Create an install-config.yaml to launch a cluster with](<https://docs.openshift.com/container-platform/4.12/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-initializing-manual_installing-platform-agnostic>)
 
-### I want to try the unreleased WMCO, and deploy it using OLM
-* [Build the operator](#build)
-* [Deploy the operator through OLM](#deploying-the-operator-through-olm)
+#### Configure Networking
```

Comment:
With this subheading, I'm trying to move this
https://github.com/openshift/windows-machine-config-operator/blob/master/docs/setup-hybrid-OVNKubernetes-cluster.md 
into the HACKING.md 
I have no issue with linking to it here, but since it would have to be done every time you create a cluster to run the tests, it makes sense to me to not have two docs for it and just include it in hacking

Reply:
The hacking doc should be succinct and not verbose. We should try and not overload it. There is history behind [setup-hybrid-OVNKubernetes-cluster.md](https://github.com/openshift/windows-machine-config-operator/blob/master/docs/setup-hybrid-OVNKubernetes-cluster.md). It was created when this was not well documented in our OpenShift docs. So even pointing to the OpenShift docs is sufficient.